### PR TITLE
clean and update code

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

File: htdocs/core/class/notify.class.php [hunk 1]
```diff
@@ -277,14 +277,11 @@ function getNotificationsArray($notifcode, $socid=0, $object=null, $userid=0, $s
...
-	 *	@param 	array	$filename_list		List of files to attach (full path of filename on file system)
-	 *	@param 	array	$mimetype_list		List of MIME type of attached files
-	 *	@param 	array	$mimefilename_list	List of attached file name in message
-	 *	@return	int							<0 if KO, or number of changes if OK
+	 * 	@param	string	$notifcode		Code of action in llx_c_action_trigger (new usage) or Id of action in llx_c_action_trigger (old usage)
+	 * 	@param	Object	$object			Object the notification deals on
+	 *	@return	int						<0 if KO, or number of changes if OK
 	 */
-	function send($notifcode, $object, $filename_list=array(), $mimetype_list=array(), $mimefilename_list=array())
+	function send($notifcode, $object)
```
Comment (eldy) [inline htdocs/core/class/notify.class.php]: Why removing the parameters ?

Comment (simnandez) [reply htdocs/core/class/notify.class.php]: @grandoc : Several parameters have been added to the send function while you edited the code. Remember to update your repositories before doing the pull request

**Comment by grandoc** [comment]
Juanjo comment was the good one, I fixed it.

**Comment by fappels** [comment]
Parameter documentation is still removed.

File: htdocs/core/class/notify.class.php [hunk 2]
```diff
@@ -277,12 +277,9 @@ function getNotificationsArray($notifcode, $socid=0, $object=null, $userid=0, $s
...
 	 *
-	 * 	@param	string	$notifcode			Code of action in llx_c_action_trigger (new usage) or Id of action in llx_c_action_trigger (old usage)
-	 * 	@param	Object	$object				Object the notification deals on
-	 *	@param 	array	$filename_list		List of files to attach (full path of filename on file system)
-	 *	@param 	array	$mimetype_list		List of MIME type of attached files
-	 *	@param 	array	$mimefilename_list	List of attached file name in message
-	 *	@return	int							<0 if KO, or number of changes if OK
+	 * 	@param	string	$notifcode		Code of action in llx_c_action_trigger (new usage) or Id of action in llx_c_action_trigger (old usage)
+	 * 	@param	Object	$object			Object the notification deals on
+	 *	@return	int						<0 if KO, or number of changes if OK
```
Comment (eldy) [inline htdocs/core/class/notify.class.php]: Some parameters disappears from the comment area

**Review by grandoc — COMMENTED** [review]
missing comments re-added

**Comment by eldy** [comment]
This PR is now too old, so i close it. A lot of code may have been validated since the PR and the PR may be obsolete, or too difficult to merge due to other changes. Best choice is too cancel it to avoid to keep PR with too low chances to be validated.

**Pull request closed without merging**